From 019272dce74c45f504975023c55b5bbedd2338b9 Mon Sep 17 00:00:00 2001
From: Peter Vary <pvary@cloudera.com>
Date: Fri, 18 Nov 2016 11:00:19 +0100
Subject: [PATCH 0826/1164] CDH-41489: HIVE-10276: Implement
 date_format(timestamp, fmt) UDF (Alex Pivovarov
 via Jason Dere)

q.out fix

Change-Id: I8a5e5ca1896c7437e57f9628c980a0665487c080
---
 .../results/clientpositive/udf_date_format.q.out   |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/ql/src/test/results/clientpositive/udf_date_format.q.out b/ql/src/test/results/clientpositive/udf_date_format.q.out
index acca21e..9b39ae4 100644
--- a/ql/src/test/results/clientpositive/udf_date_format.q.out
+++ b/ql/src/test/results/clientpositive/udf_date_format.q.out
@@ -26,11 +26,11 @@ STAGE PLANS:
         TableScan
           alias: _dummy_table
           Row Limit Per Split: 1
-          Statistics: Num rows: 0 Data size: 1 Basic stats: PARTIAL Column stats: COMPLETE
+          Statistics: Num rows: 1 Data size: 1 Basic stats: COMPLETE Column stats: COMPLETE
           Select Operator
             expressions: 'Wednesday' (type: string)
             outputColumnNames: _col0
-            Statistics: Num rows: 0 Data size: 1 Basic stats: PARTIAL Column stats: COMPLETE
+            Statistics: Num rows: 1 Data size: 93 Basic stats: COMPLETE Column stats: COMPLETE
             ListSink
 
 PREHOOK: query: --string date
-- 
1.7.9.5

